rocky/tools/templatetags/ooi_extra.py

```diff
@@ -99,3 +100,8 @@ def clearance_level(ooi: OOI) -> ScanLevel:
 @register.filter
 def ooi_type(reference_string: str) -> str:
     return Reference.from_str(reference_string).class_
+
+
+@register.filter
+def get_user(ooi: OOI) -> str:
```

Comment:
Filter name is ambiguous, you might want to rename it to something like `get_user_full_name`